src/modules/form/components/FormSelect.tsx

```diff
@@ -143,7 +145,8 @@ const FormSelect = <Multiple extends boolean | undefined>({
       groupBy={isGrouped ? (option) => option.groupLabel || '' : undefined}
       isOptionEqualToValue={isOptionEqualToValue}
       value={value}
-      {...props}
+      aria-label={ariaLabel}
```

Comment:
also has a dup on https://github.com/greenriver/hmis-frontend/pull/676/files#diff-2df75c89af9883b0513486e14ca3c47dc1b55e5ed219867025d72d005c6b4d63 that passes it to the input